packages/back-end/src/scim/middleware/scimMiddleware.ts

```diff
@@ -0,0 +1,47 @@
+import { Response, NextFunction } from "express";
+import { getAccountPlan } from "enterprise";
+import { AuthRequest } from "../../types/AuthRequest";
+import { usingOpenId } from "../../services/auth";
+
+export default function scimMiddleware(
+  req: AuthRequest,
+  res: Response,
+  next: NextFunction
+) {
+  const acceptHeader = req.get("Accept");
+
+  // Check if the Accept header specifies SCIM JSON
+  if (acceptHeader && acceptHeader.includes("application/scim+json")) {
+    res.setHeader("Content-Type", "application/scim+json");
+  }
+
+  req.checkPermissions("manageTeam");
```

Comment:
Wrap this in a try/catch and return a SCIM formatted error response instead of our default one